doc/src/modules/parsing.rst

```diff
@@ -58,19 +58,19 @@ don't need to be manually added by the user.
 
 .. autofunction:: sympy.parsing.sympy_parser.factorial_notation
 
-Experimental `\LaTeX` Parsing
------------------------------
+Experimental \LaTeX Parsing
+--------------------------------------
 
-`\LaTeX` parsing was ported from
+\LaTeX parsing was ported from
 `latex2sympy <https://github.com/augustt198/latex2sympy>`_. While functional
 and its API should remain stable, the parsing behavior or backend may change in
 future releases.
 
-`\LaTeX` Parsing Caveats
-------------------------
+\LaTeX Parsing Caveats
+---------------------------------
```

Comment:
It this change accidental?

Reply:
I don't know. Will build locally and will let you know.

Reply:
I tried to build the docs and getting the following exception,
```python
PYTHONPATH=..: sphinx-build -b html -d _build/doctrees -W src _build/html
Running Sphinx v1.6.7

Extension error:
Unknown event name: config-inited
Makefile:49: recipe for target 'html' failed
make: *** [html] Error 1
```

Reply:
It can be because of your sphinx version is old.
Can you update your sphinx version?

Reply:
Those extra `-` aren't making any difference. Will remove them in next commit.